bin/app.py

```diff
@@ -15,19 +15,23 @@ def main():
 
     args = parser.parse_args()
 
-    TARGETS_PARAMETERS = [
+    TMBA_TARGETS_PARAMETERS = [
         "Power Generation (Aggregate)",
         "Power Generation Capacity (Aggregate)",
     ]
+    IAM_TARGETS_PARAMETERS = ["Primary Energy", "Secondary Energy|Electricity"]
 
     if not args.sdf_source:
         # TODO: this is just a quick hacky/demo version of course
         assert args.materials and args.targets
         sdf = create_sdf(
             intensities=ds.MatDPDBIntensitiesSource(args.materials),
             indicators=ds.MatDPDBIndicatorsSource(args.materials),
-            targets=ds.TMBATargetsSource(
-                args.targets, TARGETS_PARAMETERS, ds.MatDPDBIntensitiesSource
+            # targets=ds.TMBATargetsSource(
```

Comment:
Remove comment?

Reply:
I thought I'll keep it as another example. Since ytou are working on it, shall I leave it to you? Probably will be removed if you add some more args anyway.